tests/CMakeLists.txt

```diff
@@ -1,6 +1,21 @@
 include(AwsTestHarness)
 enable_testing()
 
+option(BYO_CRYPTO "Don't build a tls implementation or link against a crypto interface. This feature is only for unix builds currently." OFF)
+
+set(S3_TEST_BUCKET "aws-crt-canary-bucket" CACHE STRING "Name of the bucket to use for tests" )
```

Comment:
I've been using environment variables for integration test configuration.  Cmake variables doesn't seem like a great fit.

Reply:
Moved the bucket/region CMake settings into environment variables and made that into a separate PR: https://github.com/awslabs/aws-c-s3/pull/125